sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,575 @@
+* **Commitment to transparent governance:** The community's governance over the open source ecosystem is unchanged, retaining control over all aspects of Stacks technical and ecosystem development via SIPs. The Endowment entity, free of restrictions placed on the current foundation and others, will also be able to provide a more granular level of financial reporting. The community is the key stakeholder and guide for the Endowment.
+
+Together, these changes will give the Stacks ecosystem the capital, execution speed, and organizational clarity required to compete with top ecosystems — ensuring that Stacks can continue to lead and aggressively grow market share as the leading Bitcoin L2; unlocking long-term value for builders, users, and holders.
+
+### **Stacks Roadmap Summary**
+
+This treasury will fuel development along the [Stacks roadmap](http://stacksroadmap.com), which has been refreshed via a series of [community workshops](https://forum.stacks.org/t/stacks-roadmap-draft-final-call-community-feedback/17984) ([1](https://forum.stacks.org/t/stacks-roadmap-draft-final-call-community-feedback/17984)), the latest version of which has been published in support of this SIP. The treasury will be created to deliver on these and other priorities as defined by builders and key stakeholders and provide the necessary resources for Stacks’ progression in the following critical areas, among others:
+
+**1\. Stacks Core Development:**
+
+Upgrades to increase transaction speed, reliability, scalability, and security — including improved block production, miner replay logic, Clarity-to-Wasm compilation, ongoing enhancements to PoX and stacking UX, security bounties, audits, and monitoring.
+
+**2\. sBTC Protocol & Stacks Value Accrual:**
+
+Treasury funding will also support the development of new mechanisms that drive STX demand such as dual stacking, bitcoin payments use cases, and BTC yield strategies — as well as continued development of critical sBTC features like self-custodial minting and upgrades for further trust-minimization. By funding these areas, the Endowment aims to create a self-reinforcing flywheel where protocol revenue grows treasury reserves → reserves fund development and adoption → adoption increases protocol usage and STX demand → drives continued revenue growth. This model provides a sustainable path for long-term operations and Stacks ecosystem growth.
+
+**3\. Ecosystem Growth:**
+
+Strategic efforts to scale TVL, liquidity, and user adoption through community based marketing and growth strategies, grant programs, DeFi incentives, sBTC TVL milestones, stablecoin integrations, and centralized exchange listings for SIP-10 tokens.
+
+This roadmap reflects the core areas where capital from the Stacks endowment will be deployed if SIP-031 is approved — focusing on core protocol upgrades, institutional support, developer traction, and Stacks value accrual.
```

Comment:
Wordsmithing:  please be consistent in how you spell "endowment."  Is it "Stacks endowment" or "Endowment"?